suite-native/formatters/src/components/EthereumTokenToFiatAmountFormatter.tsx

```diff
@@ -35,17 +33,17 @@ export const EthereumTokenToFiatAmountFormatter = ({
     numberOfLines,
     ...rest
 }: EthereumTokenToFiatAmountFormatterProps) => {
-    const fiatCurrency = useSelector(selectFiatCurrency);
+    const fiatCurrencyCode = useSelector(selectFiatCurrencyCode);
     const { FiatAmountFormatter } = useFormatters();
-    const fiatRateKey = getFiatRateKey(ethereumToken, fiatCurrency.label, contract);
+    const fiatRateKey = getFiatRateKey('eth', fiatCurrencyCode, contract);
     const rate = useSelector((state: any) => selectFiatRatesByFiatRateKey(state, fiatRateKey));
```

Comment:
```suggestion
    const rate = useSelector((state: FiatRatesRootState) => selectFiatRatesByFiatRateKey(state, fiatRateKey));
```

If you're going to make any changes to this PR, could you please change this? Tested locally that type check will pass with this and it's better than leaving `any`

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/8670/commits/1ae1f4f3b19ce2359717de448fea907dc4548be3